conda-forge.yml

```diff
@@ -4,3 +4,6 @@ github:
 conda_build:
   error_overlinking: true
 conda_forge_output_validation: true
+provider:
+  linux_aarch64: azure
+  linux_ppc64le: azure
```

Comment:
@lgray can you link to the docs that you read that cover providers? I'm trying to assemeble some conda-forge bookmarks for myself for the future. :grimacing:

Reply:
https://conda-forge.org/docs/maintainer/conda_forge_yml/#provider